### ISSUE TITLE
Final things to do for the ICCS paper?

### ISSUE THREAD

**Issue by Daniel-Mietchen** [fnielsen/scholia#464]
Just noticed in the project description that this is supposed to be submitted next week, so what is still to be done by then?

**Comment by egonw** [comment]
Deadline is oct 1... Website is outdated.

**Comment by egonw** [comment]
I set up a place to write the paper... I hope you're all (@fnielsen, @Daniel-Mietchen, @Chris-Evelo, @DeniseSl22) OK with using Overleaf: https://v2.overleaf.com/read/jvrsgpdrwvrj

**Comment by Chris-Evelo** [comment]
Fine with me. Except that link seems to go to a "read" version that I cannot edit

**Comment by Daniel-Mietchen** [comment]
@egonw The Overleaf link is still read only.

**Comment by egonw** [comment]
I send an invite to your Gmail.

**Comment by Daniel-Mietchen** [comment]
OK, found it.

**Comment by Daniel-Mietchen** [comment]
Submission guidelines sit at https://jcheminf.biomedcentral.com/submission-guidelines (generic) and https://jcheminf.biomedcentral.com/submission-guidelines/preparing-your-manuscript/software (specific to software papers).

How many pages are we aiming at?

**Comment by egonw** [comment]
Concise (=short) but complete, is what I'll aim for. I already requested an extension, but I do hope to find time soon to make a good headstart (first need to get far enough with my presentations this week)...

I propose a [Methodology](https://jcheminf.biomedcentral.com/submission-guidelines/preparing-your-manuscript/methodology) paper, as I want this paper to propose a more FAIR way of disseminating chemical knowledge.

**Comment by Daniel-Mietchen** [comment]
Maybe it's time to close this now?

**Comment by egonw** [comment]
No, please keep open.

**Comment by Daniel-Mietchen** [comment]
OK, then maybe we need a new timeline?

**Comment by egonw** [comment]
Sure. I suggest ASAP. I had hoped to send a draft by the end of the xmas holidays, but those did not go as planned :/ Neither did the past year really :/

**Comment by Daniel-Mietchen** [comment]
The Bacting paper is up at https://doi.org/10.21105/joss.02558 .

**Comment by egonw** [comment]
I'm picking up this paper next week, now that the Bacting paper is out.

**Comment by Chris-Evelo** [comment]
Really nice to see that paper out. I quickly checked it and I think it is a nice and clear "update to next generation" description for Bioclipse users. I wonder whether you also need a new introduction about why people need Bacting or Bioclipse in the first place. Like a video that shows what you can do with it? Or is that already there, and maybe I did not read it well enough to see it mentioned?

**Comment by egonw** [comment]
Yea, good question and I'm not sure. For now, I needed it to be able to cite this as "SOP" for other papers. Making a video costs a lot of time, and more something for people that still have time for us, or until such videos are recognized as important research output and we actually get paid to make them...

**Comment by Chris-Evelo** [comment]
Yes, that is always a problem. Although many projects nowadays at least have a budget for dissemination.

Maybe Lauren can help? She is in fact paid for such things. But of course, she would need an introdiction.

**Comment by egonw** [comment]
New location of the writing: https://www.overleaf.com/project/63e8c99dcd80245076e601bb

**Comment by egonw** [comment]
New draft location: https://www.overleaf.com/read/rvyvybbznwzp

**Comment by Chris-Evelo** [comment]
I don't really understand why we have a "draft" and a "writing" location

I am also not sure where to add comments.

For now:
1) I fixed your email address
2) I like the abstract, except the last line. The part before introduces chemistry on the web, then WikiData then Scholia as a way to use it. But the last line goes back to Wikidata? That should either be moved up, or clarified how it relates to Scholia work done.

**Comment by egonw** [comment]
> Deadline is oct 1... Website is outdated.

That did not age well :)
